pages/static/vendor/js/djaodjin-sidebar-gallery.js

```diff
@@ -48,64 +23,74 @@
             </div>\
         </div>";
 
-    var id = null;
-    var count = 0;
-
     function SidebarGallery(options){
         this.options = options;
-        this._init();
+        this.init();
         Dropzone.autoDiscover = false;
     }
 
-    function notifyUser(descr, level){
-        $("body").prepend("<div class=\"notify-user-label " + level + "-label\">" + descr + "</div>");
-        setTimeout(function(){
-            $(".notify-user-label").remove();
-        }, 5000);
-    }
+    SidebarGallery.prototype = {
+        init: function(){
+            sidebarGallery = this;
+            sidebarGallery.originalTags = [];
+            sidebarGallery.selectedMediaLocation = "";
+            sidebarGallery.currentInfo = "";
+            sidebarGallery.selectedMedia = null;
+            sidebarGallery.initSidebar();
+            sidebarGallery.initDocument();
+            sidebarGallery.initDropzone();
+        },
 
-    function update_progress_info(url) {
-        $.getJSON(url, {'X-Progress-ID': id}, function(data, status){
-            if(data){
-                $("#progress-span").text((data.uploaded / data.lenght) * 100);
-            }
-            else{
-                $("#progress-span").text(100);
-                return false;
-            }
-            setTimeout(update_progress_info(url), 100);
-        });
-    }
+        initSidebar: function(){
+            $("body").append(sidebarHtml);
+            $("body").append(sidebarGallery.options.toggle);
+        },
 
-    SidebarGallery.prototype = {
+        initDocument: function(){
+            $(document).on("click", "#delete_media", sidebarGallery.deleteMedia);
```

Comment:
Do all of these handlers need to be on the top-level document? Could they not be attached to a more narrow part of the DOM?